microsite/data/plugins/opencost.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: OpenCost
+author: Matt Ray
+authorUrl: https://opencost.io
+category: Monitoring
```

Comment:
"Kubecost" & "Cost Insights" are in "Discovery" - I'm happy with "Monitoring" think it makes sense:)

Reply:
I'd prefer "Monitoring" since we're only reporting costs with different views.